src/_guides/libraries/futures-error-handling.md

```diff
@@ -1,19 +1,24 @@
 ---
-layout: article
+layout: default
 title: "Futures and Error Handling"
 description: "Everything you wanted to know about handling errors and exceptions when working with Futures (but were afraid to ask)."
-written: 2013-03-07
-updated: 2013-05-30
-category: libraries
 ---
 
-_Written by Shailen Tuli <br>
-March 2013 (updated May 2013)_
-
-This article covers the subject of error handling when dealing with Futures.
-If you are unfamiliar with the general concepts behind Futures, we
-recommend you first read
-[Asynchronous Programming: Futures](/tutorials/language/futures).
+In many cases, when you want to write asynchronous code in Dart,
+you can use these built-in [`async` functions and `await`
+expressions](/guides/language/language-tour#asynchrony-support)
+where you handle errors using `try-catch` and `throw`.
+However, sometimes you need more control than what you can
+accomplish with the built-in features. [PENDING: example?]
+
+In this situation, you can use the
+[Future API,](/guides/libraries/library-tour#future) part of the
+[`dart:async`](https://api.dartlang.org/stable/dart-async/dart-async-library.html)
+core library. A `Future` object represents a potential value, or error,
```

Comment:
It seems weird to introduce Future here, when you've already talked about the Future API.